plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -149,6 +149,7 @@ def hasCapability(self, capability):
             ManagerInterface.Capability.kPublishing,
             ManagerInterface.Capability.kRelationshipQueries,
             ManagerInterface.Capability.kExistenceQueries,
+            ManagerInterface.Capability.kDefaultEntityReferences,
```

Comment:
Q: The feature appears to be 'singular' defaultEntityReference, while this capability is plural. Intended?

Reply:
I don't think we really thought about it, other than to say that the mapping from capability to API methods is, in general, a one-many relationship. In this particular case there is only one API method associated with the capability. But e.g. `kPublishing` and `kRelationshipQueries` each have two associated API methods.